### ISSUE TITLE
Convergence of the training

### ISSUE THREAD

**Issue by michaelzhang917** [metalbubble/TRN-pytorch#15]
Hi I have used the following script to run the training on the jester but it looks like the accuracy does't improve after several epochs. Could you provide me the hyperparameter you used for the provided pretrained model.
  
CUDA_VISIBLE_DEVICES=0,1 python main.py jester RGB \
                     --arch BNInception --num_segments 8 \
                     --consensus_type TRNmultiscale --batch-size 64

In addition, I could not resume the training from the pretrained model using

CUDA_VISIBLE_DEVICES=0,1 python main.py jester RGB \
                     --arch BNInception --num_segments 8 \
                     --consensus_type TRNmultiscale --batch-size 64
                     --resume pretrain/TRN_jester_RGB_BNInception_TRNmultiscale_segment8_best.pth.tar

It printed out the following message and then quit
=> loading checkpoint 'pretrain/TRN_jester_RGB_BNInception_TRNmultiscale_segment8_best.pth.tar'
=> loaded checkpoint 'False' (epoch 120)
video number:118562
video number:14787
group: first_conv_weight has 1 params, lr_mult: 1, decay_mult: 1
group: first_conv_bias has 1 params, lr_mult: 2, decay_mult: 0
group: normal_weight has 83 params, lr_mult: 1, decay_mult: 1
group: normal_bias has 83 params, lr_mult: 2, decay_mult: 0

But I have successfully resume training from my own checkpoint.

**Comment by michaelzhang917** [comment]
It resolved after I set epochs number to 200 since the original model has trained 120 epochs and the default number of epochs is 120 as well.

**Comment by ldtuanAPCS** [comment]
Hi @michaelzhang917 , did you reproduced the authors' result for jester dataset? Or you just continued training on the jester pretrained model? Thanks.
